File: DrawingServer/DrawnigContracts/DTO/MarkersDTO/markerDataDto/Marker.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DrawnigContracts.DTO.MarkersDTO.markerDataDto
{
    public class Marker
    {
        public MarkerColor markerColor { get; set; }
        public string markerId { get; set; }
        public MarkerLocation markerLocation { get; set; }
        public string markerType { get; set; }
        public Point originScreen { get; set; }
    }
}

File: DrawingClient/src/app/DTO/DATA/Marker.ts
import { MarkerColor } from './marker-color';
import { MarkerLocation } from './marker-location';
import { Point } from './point';
export class Marker {
    markerColor: MarkerColor
    markerType: string
    markerLocation: MarkerLocation
    markerId: string = null;
    originScreen:Point
}
